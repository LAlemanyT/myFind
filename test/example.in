./myfind.py test --regex=".in$"
