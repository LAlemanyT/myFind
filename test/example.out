test/noDirectory.in
test/noArguments.in
test/nameAndFlags.in
test/flagsAndRegexWithSpaces.in
test/relativeAndFlags.in
test/operatingWithFlags.in
test/relativePath.in
test/invalidCommandNoPath.in
test/fileDoesntExist.in
test/example.in
test/tooManyArguments.in
test/regexForStart.in
